services/provider/server/server.go

```diff
@@ -427,6 +427,17 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	subscriptionSpec, err := s.getOCSSubscriptionSpec(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get subscription %v", err)
+	}
+
+	extR = append(extR, &pb.ExternalResource{
+		Name: "ocs-operator",
```

Comment:
This name does not make sense, are we going to create/update a subscription called "ocs-operator" on the client side? This should be the name of the subscription you want to create/update on the client side